web-ui/view/index.ml

```diff
@@ -32,3 +32,114 @@ let render github gitlab =
           else li [ txt "No GitLab organisations" ]);
         ];
     ]
+
+let rows prefix orgs =
+  let open Organisation.Make (struct
+    let prefix = prefix
+  end) in
+  let orgs = List.sort String.compare orgs in
+  List.fold_left (fun l org -> List.append l [ row ~org ]) [] orgs
```

Comment:
```suggestion
  List.map (fun org -> [ row ~org ]) orgs
```
Isn't it a map here?

Reply:
I think that your suggestion ends up giving us a list of lists and so we'd need to flatten that